src/main/webapp/style.scss

```diff
@@ -4,16 +4,47 @@
 
 /* Define color scheme of page */
 :root {
-  --error: red;  
+  --error: red; 
+  --gray: #D3D3D3; 
+  --btn-color: #B590CA;
+  --box-bg: #DDF3F5;
+  --dark-gray: #5D5B6A;
+  --disabled-txt: #999;
 }
 
-h1 {
+h2, h4 {
   margin: 0 auto;
 }
 
+.search-title {
+  margin-bottom: 20px;
+}
+
+button {
+  background-color: var(--gray);
+  color: black;
+    font-size: 16px;
```

Comment:
Fix indentation here.

Reply:
Also `outline: none` removes the weird black outline we see when the button is clicked.